content/confirmation/workshop.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Thank You | Workshop
+meta_desc: This page is provides confirmation about register for a Pulumi workshop.
+type: page
+layout: confirmation
+url: /confirmation/workshop
```

Comment:
Is the `url` parameter necessary? I think we might be able to delete this line.

```suggestion
```

Reply:
I guess it's not. I removed it.